RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Release To The Google Play Store
+
+## Setup
+
+Run the setup script:
+
+```
+> bin/setup
+```
+
+## Releasing the app
+
+```
+> fastlane android release
```

Comment:
I have this zsh alias that allows me to copy `$` and `%` prompts without problem:

```sh
alias \$=\;
alias \%=\;
```

`%`, `$`, and `#` are commonly understood in the grumpy old unix world to mean "this is to be run at your shell" (`#` means "this is to be run with elevated privileges"). So I'd pick one of those if you're going to show the prompts.